src/cljs/replumb/repl.cljs

```diff
@@ -499,6 +500,40 @@
                                 (common/wrap-error error)
                                 (common/wrap-success nil)))))))))))))
 
+(defn fetch-source
+  [{:keys [verbose read-file-fn!]} var paths-to-try cb]
+  (load/read-files-and-callback! verbose
+                                 paths-to-try
+                                 read-file-fn!
+                                 (fn [result]
+                                   (let [source (:source result)
+                                         rdr (rt/source-logging-push-back-reader source)]
+                                     (dotimes [_ (dec (:line var))] (rt/read-line rdr))
+                                     (-> (r/read {:read-cond :allow :features #{:cljs}} rdr)
+                                         meta 
+                                         :source
+                                         common/wrap-success
+                                         cb)))))
+
+(defn process-source
+  [opts cb data env sym]
+  (let [var (get-var opts env sym)
+        cb-partial (partial call-back! (merge opts {:no-pr-str-on-value true}) cb data)
+        call-back (fn [result]
+                    (if (nil? result)   ; if nil? means that no file was found
+                      (cb-partial (common/wrap-success "nil"))
+                      (cb-partial result)))]
```

Comment:
The else branch should already receive a wrapped result, see lines above (`->`..)
But yes, maybe better wrap in one place.